modules/storages/app/common/storages/peripherals/oauth_configurations/one_drive_configuration.rb

```diff
@@ -34,13 +34,14 @@ module OAuthConfigurations
       class OneDriveConfiguration < ConfigurationInterface
         DEFAULT_SCOPES = %w[offline_access files.readwrite.all user.read sites.readwrite.all].freeze
 
-        attr_reader :oauth_client
+        attr_reader :oauth_client, :oauth_uri
 
         def initialize(storage)
+          super()
```

Comment:
Unnecessary call to `super`. There's no initialization there.

Reply:
Well, if I do not put it here, I need to disable the linter rule. Is it really prettier?